system76/darp6/default.nix

```diff
@@ -0,0 +1,26 @@
+# Hardware profile for the Darter Pro 6 laptop by System76.
+#
+# https://system76.com/laptops/darter
+
+{ lib, ... }:
+{
+  imports = [
+    ../.
+    ../../common/pc/laptop
+    ../../common/pc/laptop/ssd
+  ];
+
+  hardware.system76 = {
+    enable = lib.mkDefault true;
+
+    # darp6 needs system76-acpi-dkms, not system76-dkms:
+    #
+    # [1] https://github.com/pop-os/system76-dkms/issues/39
+    # jackpot51> system76-acpi-dkms is the correct driver to use on the darp6
+    #
+    # system76-io-dkms also appears to be loaded on darp6 with Pop!_OS, and
+    # system76-dkms does not, and in fact refuses to load.
+    drivers.system76-acpi.enable = lib.mkDefault true;
+    drivers.system76-io.enable = lib.mkDefault true;
```

Comment:
This module option seems to be true for all modules. Just remove the option and include the driver by default.